usagov/vote-gov-drupal
# VOTE-3172 remove navajo font from votegov theme

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-3172

## Description

Remove navajo font from votegov theme

## Deployment and testing

### Post-deploy steps

1. run `npm run build` in the votegov theme
2. run `lando retune`

### QA/Testing instructions

1. visit http://vote-gov.lndo.site/nv/about-us and confirm that the navajo font is not in use and that the fonts match the rest of the site

## Checklist for the Developer




## Checklist for the Peer Reviewers
